README.md

```diff
@@ -22,28 +22,60 @@ Our project consists of two main parts. The first part of our project is data vi
 * Maeve Murphy
 * Reagan Smith
 
-## MVP objective
+## Architecture
+
+All data processing is completed using Python 3. Ruby will use the Python scripts to display the resulting graphs on web pages, titled "Overall Distribution" and "Individual Statistics."
+
+### Database Design
+
+The database is implemented using SQLite, and follows this structure:
```

Comment:
Is there supposed to be a picture here?

Reply:
I also think once we add the picture we might want a few more words to describe it.